var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -83,11 +85,13 @@ class Warpx(CMakePackage):
     variant('tprof', default=True,
             description='Enable tiny profiling features')
 
+    depends_on('sensei', when='+sensei')
```

Comment:
Let's push this to
```suggestion
    depends_on('sensei@4.0.0:', when='+sensei')
    conflicts('+sensei', when='@:22.06',
              msg='WarpX supports SENSEI 4.0+ with 22.07 and newer')
```
since this is the version range we require since https://github.com/ECP-WarpX/WarpX/pull/3211

Reply:
added the conflict now